### PR TITLE
Add even more info on loaded fonts

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0e19a5ea3193253844e78cee1f47e53e3f72ba5f: [**arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-arm-neon.apk)
- [x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-x64.apk)
- [x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-x86.apk)
- [**all access arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-arm64.apk)
- [all access arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-arm-neon.apk)
- [all access x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-x64.apk)
- [all access x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5432-0ca716f29f5d5b9d804a398ba08c0105737fd256-android-x86.apk)
</details>

**Review by mohsenD98 — COMMENTED** [review]
can we extract the common logic into a separate function ? 

```C++
void registerFont(const QString &fontFile) {
    const int id = QFontDatabase::addApplicationFont(QDir::cleanPath(fontDir.path() + QDir::separator() + fontFile));
    if (id >= 0) {
        qInfo() << .. ;
        qInfo() << .. ;
    } else {
        QgsMessageLog:: ... 
    }
}
```

File: src/core/qgismobileapp.cpp [hunk 1]
```diff
@@ -245,8 +245,12 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
       for ( const QString &fontFile : fontFiles )
       {
         const int id = QFontDatabase::addApplicationFont( QDir::cleanPath( fontDir.path() + QDir::separator() + fontFile ) );
-        qInfo() << QStringLiteral( "App-wide font registered: %1" ).arg( QDir::cleanPath( fontDir.path() + QDir::separator() + fontFile ) );
-        if ( id == -1 )
+        if ( id >= 0 )
```
Comment (mohsenD98) [inline src/core/qgismobileapp.cpp:248]: here.

File: src/core/qgismobileapp.cpp [hunk 2]
```diff
@@ -774,8 +778,12 @@ void QgisMobileapp::readProjectFile()
     for ( const QString &fontFile : fontFiles )
     {
       const int id = QFontDatabase::addApplicationFont( QDir::cleanPath( fontDir.path() + QDir::separator() + fontFile ) );
-      qInfo() << QStringLiteral( "Project font registered: %1" ).arg( QDir::cleanPath( fontDir.path() + QDir::separator() + fontFile ) );
-      if ( id == -1 )
+      if ( id >= 0 )
```
Comment (mohsenD98) [inline src/core/qgismobileapp.cpp:781]: and here.

**Pull request closed without merging**